src/py/flwr/simulation/app.py

```diff
@@ -225,7 +231,7 @@ def run_simulation_process(  # pylint: disable=R0914, disable=W0212, disable=R09
                 )
             backend_config: BackendConfig = fed_opt.get("backend", {})
             verbose: bool = fed_opt.get("verbose", False)
-            enable_tf_gpu_growth: bool = fed_opt.get("enable_tf_gpu_growth", True)
+            enable_tf_gpu_growth: bool = fed_opt.get("enable_tf_gpu_growth", False)
```

Comment:
I left it `true` on purpose with the idea of making things easier for those that want to use TF simulations w/ GPU. They won't need to manually set the env variable. What do you think makes more sense to do?

Reply:
Am I right in understanding that this `enable_tf_gpu_growth` is set in `pyproject.toml` like so?
```toml
[tool.flwr.federations.local-simulation]
# enable-tf-gpu-growth = True
```

If so, can we set it to `False` in code, and then the line above commented out in case users want to include it?

Reply:
as discussed, let's keep it as `False` and revisit this topic in a new PR.